doc/development.md

```diff
@@ -10,6 +10,14 @@ sudo apt-get install libblkid-dev
 sudo dnf install libblkid-devel
 ```
 
+## Generate
+
+Install [schematyper](https://github.com/idubinskiy/schematyper) to generate `schema.go` from the Ignition JSON schema.
```

Comment:
Want to include the full path to the file (`config/types/schema.go`), and maybe explain how/why it's generated (schema is stored in [json schema] format for interoperability, is autogenerated into go structs stored in config/types/schema.go, so don't edit that by hand)?